go.mod

```diff
@@ -44,3 +44,4 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	honnef.co/go/tools v0.2.0 // indirect
 )
+replace github.com/pingcap/kvproto => /home/xiaojin.zxj/010.tidb/kvproto
```

Comment:
Please use the repo of your own instead of the local path. The other couldn't test it with the local path.